projects/module-1/ideabox-group.md

```diff
@@ -2,326 +2,235 @@
+  - your team's GitHub repo
+  - the GitHub Pages deployed site
+    - Instructions for deploying: Repo Settings >> Pages >> Select the `main` branch as your Source branch >> Save >> Send us the URL they give you in the pop up that says "Your site is ready to be published at [URL]"  
 
-As a user,
-- When I create one idea successfully, then manually refresh the page,
-- The idea object is still saved in the ideas array
-- The idea card is still visible on the DOM
+Then, as a team:
+* Have all team members clone down the repo.  
 
-As a user,
-- When I create two cards successfully, delete one, then refresh the page,
-- One idea object is still in the ideas array (the one I did not delete)
-- One idea card is still visible on the DOM (the one I did not delete)
+You're ready to start iteration 0!
+</section>
 
-As a user,
-- When I favorite an idea card, then refresh the page,
-- That idea card is still in the "favorite" state with the filled in star icon (in both the ideas array and on the DOM)
+<section class="answer">
+### Iteration 0 - Basic Layout
 
-### Commenting on Ideas
+![iteration 0](./assets/ideaBox/iteration0.png)
 
-This extension adds the ability for your user to add comments to an idea.
+- When the page loads, we should see:
+  -  a title, two inputs, and a button
+  -  a page layout and colors that match the comp above
 
-#### Data Model
+- When a user clicks "Save," a new idea object should be created and logged to the console. The idea object should have:
+  - title
+  - body
+  - id
 
-* The idea objects should now include a new property: `comments`. This will begin as an empty array.  
-* You will need to write functions that allow you to add new comments to an idea's `comments` array.  
-* A comment will be an object with the following properties: `id` (should be unique), `content` (a string of the comment itself).
+- Every time a new idea is created, it should be added to a list that represents your data model.
+</section>
 
+<section class="answer">
+### Iteration 1 - Adding Ideas
 
-As a user,
-- When I click the "Add" icon on an idea card,
-- A form to add a comment appears on that idea card
+![iteration 1](./assets/ideaBox/iteration1.png)
 
-As a user,
-- When I open the comment form on a card, type something in, and click "Add Comment",
-- The text typed in is now a comment attached to this card
+- When a user fills out both inputs and clicks "Save": 
+  - a new idea should be added to the data model.
+  - a card should appear on the screen to match the comp above. The card should show the title and body the user entered into the form.
+  - the form inputs should clear out to be ready for the next idea.
 
-As a user,
-- When I open the comment form on a card, type something in, and click "Add Comment",
-- The "Comment" input field clears out and is ready to accept another comment
+- When a user clicks "Save" with an empty Title or Body input field:
```

Comment:
This note makes it sound like when the Save button is clicked, then the button should become disabled, but I think you mean it should be disabled even before they try to click it.

Reply:
fixed